packages/cozy-intent/src/api/services/StaticService.ts

```diff
@@ -34,6 +38,14 @@ export const StaticService = {
     return nativeEvent.data.includes(Strings.postMeSignature)
   },
 
+  getBarInitAPI(): ((webviewContext: WebviewService) => void) | undefined {
+    try {
+      return cozy?.bar?.setWebviewContext
+    } catch (err) {
+      return undefined
```

Comment:
I think we should definitely log this error. No?

Reply:
I guess so yes, btw this error should only happen in dev mode when webpack for some reason fails to reach the cozy bar module (happened only once to me, and not sure why it happened)